docs/man/docker-run.1.md

```diff
@@ -230,10 +229,6 @@ which interface and port to use.
 **--lxc-conf**=[]
    (lxc exec-driver only) Add custom lxc options --lxc-conf="lxc.cgroup.cpuset.cpus = 0,1"
 
-**--log-driver**="|*json-file*|*syslog*|*none*"
```

Comment:
Because these are not valid for docker run or docker create.  They are only valid for docker -d.  There are lots of others there, which should be removed.  Docker run just ignores these values.